pkgs/by-name/ru/ruff/package.nix

```
@@ -27,8 +27,8 @@ python3Packages.buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "astral-sh";
     repo = "ruff";
-    rev = "refs/tags/${version}";
-    hash = "sha256-N3TplR+vPu2r56VP/vnOfkxN3Lh2o92kE2hFZKLXO04=";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```
we should keep that to not confuse branches with tags if they are named the same

---

Indeed, my plan was to switching to the newly introduced `tag` option. Now I have fixed it.

---

I didn't know that was a thing

---

Literally from yesterday ;)
https://github.com/NixOS/nixpkgs/pull/355973